src/azure-cli-core/azure/cli/core/tooling.py

```diff
@@ -0,0 +1,194 @@
+"""tooling integration"""
```

Comment:
We don't require file level doc string. If you'd like to add, please move it behind the license header and add more details.